test/java/skip.java

```diff
@@ -0,0 +1,20 @@
+class Test {
```

Comment:
Is this file associated to a test ?

Reply:
It is used by these two tests: https://github.com/mozilla/grcov/pull/416/files#diff-35517b643f4f9034bdad1c30fde730adR1361

Reply:
Sorry but I don't see any test in this patch. Or maybe I missed something...

Reply:
The tests are right at the end of the collapsed diff for src/path_rewriting.rs